demo/python/image_classification.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+
+import cv2
+from mmdeploy_python import Classifier
```

Comment:
`mmdeploy_python` will not be added to the python environment when building from the source.

Reply:
PYTHONPATH should be set. I am going to note it down in PR #531